src/js/bootstrap-slider.js

```diff
@@ -1060,9 +1060,14 @@ const windowIsDefined = (typeof window === "object");
 				return this;
 			},
 
-			refresh: function() {
+			refresh: function(options) {
+				var currentValue = this.getValue();
```

Comment:
actually thinking about this in further detail, this may be over-engineering on my part. curious to hear your thoughts here

Reply:
Well, I tried some stuff in **node**

```js
> x = 5
> x + 5
10
> x.useCurrentValue === true
false
> (x + 5).useCurrentValue === true
false
> x = 'abc'
> x.useCurrentValue === true
false
> x = null
> x.useCurrentValue === true
TypeError: Cannot read property 'useCurrentValue' of null
```

Everything is an object? So I guess `.useCurrentValue` would work without having to check if `options` is an object.

Reply:
> can we call this method inside the `if` statement so that it is only invoked when we need it?
> 
> Also lets make it a `const` instead of a `var` (in general, favor ES6 variable declarations since they scope to the nearest enclosing block)

I can't the `getValue()` nest this inside an if statement because I have to retrieve the current value of the slider.

Reply:
I passed a number as `options` and stepped through the debugger:

```js
$('ex1').slider(1000);
// ...
var val = options[optName];  // 1000['id'] returns `undefined`
```

![slider options 1000](https://user-images.githubusercontent.com/5477929/50674445-52d91880-0fb4-11e9-8ef9-688c9137f240.png)